addons/l10n_jo/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,413 @@
+                        <field name="name">8. Exported Sales</field>
+                        <field name="code">JO_STD_SALES_8</field>
+                        <field name="expression_ids">
+                            <record id="tax_report_vat_sale_export_zero_tag" model="account.report.expression">
+                                <field name="label">balance</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">8. Exported Sales (Base)</field>
+                            </record>
+                            <record id="tax_report_vat_sale_export_zero_tag_tax" model="account.report.expression">
+                                <field name="label">taxbalance</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">8. Exported Sales (Tax)</field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="tax_report_vat_sales_no_tax_zero" model="account.report.line">
+                        <field name="name">9. Sales that are not subject to tax</field>
+                        <field name="code">JO_STD_SALES_9</field>
+                        <field name="expression_ids">
+                            <record id="tax_report_vat_sale_export_no_tax_zero_tag" model="account.report.expression">
+                                <field name="label">balance</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">9. Sales that are not subject to tax (Base)</field>
+                            </record>
+                            <record id="tax_report_vat_sale_export_no_tax_zero_tag_tax"
+                                    model="account.report.expression">
+                                <field name="label">taxbalance</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">9. Sales that are not subject to tax (Tax)</field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="tax_report_vat_sales_exempt_local_zero" model="account.report.line">
+                        <field name="name">10. Exempted Local Sales</field>
+                        <field name="code">JO_STD_SALES_10</field>
+                        <field name="expression_ids">
+                            <record id="tax_report_vat_sale_export_exempt_local_zero_tag"
+                                    model="account.report.expression">
+                                <field name="label">balance</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">10. Exempted Local Sales (Base)</field>
```

Comment:
@ygu-odoo Taxes file looks great but for the reports to be updated, the tag name needs to be changed here as well normally.

Reply:
Sorry, it was adapted in the branch...